PSReadLine/Options.cs

```diff
@@ -271,17 +271,64 @@ public static void RemoveKeyHandler(string[] key)
         /// </summary>
         public static IEnumerable<PowerShell.KeyHandler> GetKeyHandlers()
         {
-            return GetKeyHandlers(includeBound: true, includeUnbound: false);
+            return GetKeyHandlers(includeBound: true, includeUnbound: false, Chord: null);
         }
 
         /// <summary>
-        /// Helper function for the Get-PSReadLineKeyHandler cmdlet.
+        /// Return bound/unbound key handlers without filtering.
         /// </summary>
         /// <returns></returns>
         public static IEnumerable<PowerShell.KeyHandler> GetKeyHandlers(bool includeBound, bool includeUnbound)
+        {
+            return GetKeyHandlers(includeBound: includeBound, includeUnbound: includeUnbound, Chord: null);
+        }
+
+        /// <summary>
+        /// Helper function for the Get-PSReadLineKeyHandler cmdlet.
+        /// </summary>
+        /// <returns></returns>
+        public static IEnumerable<PowerShell.KeyHandler> GetKeyHandlers(bool includeBound, bool includeUnbound, string[] Chord)
         {
             var boundFunctions = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
 
+            if (Chord != null)
+            {
+                foreach (var Key in Chord)
+                {
+                    var consoleKeyChord = ConsoleKeyChordConverter.Convert(Key);
+                    var firstKey = PSKeyInfo.FromConsoleKeyInfo(consoleKeyChord[0]);
+                    if (_singleton._dispatchTable.TryGetValue(firstKey, out var entry))
+                    {
+                        if (consoleKeyChord.Length == 1)
+                        {
+                            yield return new PowerShell.KeyHandler
+                            {
+                                Key = firstKey.KeyStr,
+                                Function = entry.BriefDescription,
+                                Description = entry.LongDescription,
+                                Group = GetDisplayGrouping(entry.BriefDescription),
+                            };
+                        }
+                        else
+                        {
+                            var secondKey = PSKeyInfo.FromConsoleKeyInfo(consoleKeyChord[1]);
+                            if (_singleton._chordDispatchTable.TryGetValue(firstKey, out var secondDispatchTable) &&
+                                secondDispatchTable.TryGetValue(secondKey, out entry))
+                            {
+                                yield return new PowerShell.KeyHandler
+                                {
+                                    Key = firstKey.KeyStr + "," + secondKey.KeyStr,
+                                    Function = entry.BriefDescription,
+                                    Description = entry.LongDescription,
+                                    Group = GetDisplayGrouping(entry.BriefDescription),
+                                };
+                            }
+                        }
+                    }
+                }
```

Comment:
Key bindings in VI mode are not covered by this. Can you please add the support to VI mode too?